docs/src/forc/plugins.md

```diff
@@ -0,0 +1,37 @@
+# Plugins
+
+The Fuel ecosystem has a few plugins which can be easily installed via cargo, and anyone can write their own plugins.
```

Comment:
Perhaps we could start with something like:

_Plugins can be used to extend forc with new commands that go beyond the native commands mentioned in the previous chapter. While the Fuel ecosystem provides a few commonly useful plugins (`forc-fmt`, `forc-lsp`, `forc-explore`), anyone can write their own!_